systori/apps/project/api.py

```diff
@@ -105,6 +143,31 @@ def search(self, request):
 
         return JsonResponse({"projects": projects})
 
+    @swagger_auto_schema(method="GET", responses={200: NoteSerializer(many=True)})
+    @swagger_auto_schema(method="POST", request_body=NoteSerializer, responses={201: NoteSerializer()})
+    @action(methods=["GET", "POST"], detail=True)
+    def notes(self, request, pk=None):
+        project = self.get_object()
+        if request.method.lower() == "get":
```

Comment:
why not `if request.method == "GET":` ?

Reply:
I searched for examples on this and that's how they were. So I assumed that the `request.method` may contain both `GET` and `get`.

Does python allow `GET` == `get`?